helm/plater/values.yaml

```diff
@@ -28,24 +28,33 @@ neo4j:
     failureThreshold: 3
     timeoutSeconds: 2
     periodSeconds: 10
-    tcpSocket:
-      port: 7687
+    exec:
+      command:
+        - bash
+        - -c
+        - IFS="/" read -a auth <<< $NEO4J_AUTH ; cypher-shell -u ${auth[0]} -p ${auth[1]} "match (c) return count(c)"
```

Comment:
i am concerned of the impact of this query on the DB when there is a boat load of records.

perhaps there is a less costly query that could be run here. like maybe "match (n:Concept) return 1 limit 1"

Reply:
yeah wasn't sure if count would be impactful, thought it was going to be a simple lookup , good call will update the query to something lighter